Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -1671,6 +1674,12 @@ extension PIF.PlatformFilter {
     public static let webAssemblyFilters: [PIF.PlatformFilter] = [
         .init(platform: "wasi"),
     ]
+
+    /// VisionOS platform filters.
+    public static let visionOSFilters: [PIF.PlatformFilter] = [
+        .init(platform: "visionos"),
```

Comment:
This needs to add "xros" or Xcode won't pick up the filters correctly. For example:

```swift
public static let visionOSFilters: [PIF.PlatformFilter] = [
    .init(platform: "xros"),
    .init(platform: "xros", environment: "simulator"),
    .init(platform: "visionos"),
    .init(platform: "visionos", environment: "simulator")
]
```